x-pack/solutions/security/plugins/security_solution/common/detection_engine/rule_management/to_simple_rule_schedule.test.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { toSimpleRuleSchedule } from './to_simple_rule_schedule';
+
+describe('toSimpleRuleSchedule', () => {
+  it.each([['10s'], ['3m'], ['5h']])('returns interval "%s" without changes', (interval) => {
+    const result = toSimpleRuleSchedule({
+      interval,
+      from: 'now-10h',
+      to: 'now',
+    });
+
+    expect(result?.interval).toBe(interval);
+  });
+
+  it.each([
+    ['now-20s', '10s', '10s'],
+    ['now-30m', '10m', '20m'],
+    ['now-3h', '1h', '2h'],
+    ['now-120s', '1m', '1m'],
+    ['now-180m', '2h', '1h'],
+    ['now-7200s', '90m', '30m'],
+  ])('returns lookback (from=%s and interval=%s)', (from, interval, expected) => {
+    const result = toSimpleRuleSchedule({
+      interval,
+      from,
+      to: 'now',
+    });
+
+    expect(result?.lookback).toBe(expected);
+  });
```

Comment:
I'd expect these tests to check what the function returns as a whole in what cases, rather than what values are returned for the individual fields such as `interval` and `lookback`. Test cases would be similar to what I tested manually and described in https://github.com/elastic/kibana/pull/204317#issuecomment-2576140982.